contracts/src/Teleporter/ITeleporterMessenger.sol

```diff
@@ -227,14 +227,14 @@ interface ITeleporterMessenger {
      */
     function getFeeInfo(
         bytes32 destinationBlockchainID,
-        uint256 messageID
+        bytes32 messageID
     ) external view returns (address, uint256);
 
     /**
-     * @dev Gets the next message ID to be used for a given chain ID.
-     * @return The next message ID to be used to send a message to the given chain ID.
+     * @dev Gets the message ID to be used for the next message sent from the contract instance.
```

Comment:
Can you add a comment here explaining that the return value is not only guaranteed to match the message ID of the next message sent to the contract, regardless of the sender address, or recipient chain or address?

Reply:
I don't think that's accurate. The return value is guaranteed to match the message ID of the next message sent from the contract. I'm guessing what you're referring to is that a caller of this method may not be guaranteed to be the next sender of a message and thus may get a different message ID assigned to their message, but I think that's assuming a specific use case that doesn't need to be documented in the contract itself.

For instance, a contract could call `getNextMessageID`, and then `sendCrossChainMessage` in the same function such that it does know it is the next caller of `sendCrossChainMessage`. I'm not sure what the use case would be, but something along the lines of:

```
contract MyContract {
    function calculateReportSend() public {
        // Calculate the message ID before sending.
        bytes32 expectedMessageID = teleporterMessenger.getNextMessageID();
        
        // Logic here to do something with expectedMessageID.
        
        // Send the message
        bytes32 messageID = teleporterMessenger.sendCrossChainMessage(...);
        require(expectedMessageID == messageID);
    }
}

Reply:
Yup, that's what I meant, thanks. In my original comment, an extraneous "not" snuck its way in. What I meant to say was: 

Can you add a comment here explaining that the return value is ~~not~~ only guaranteed to match the message ID of the next message sent to the contract, regardless of the sender address, or recipient chain or address?